cmd/beacon-chain/flags/base.go

```diff
@@ -31,6 +31,23 @@ var (
 			"Boost is an additional percentage to multiple local block value. Use builder block if: builder_bid_value * 100 > local_block_value * (local-block-value-boost + 100)",
 		Value: 10,
 	}
+	// MinBuilderBid sets an absolute value for the builder bid that this
+	// node will accept without reverting to local building
+	MinBuilderBid = &cli.Uint64Flag{
```

Comment:
+1 on adding this on the client side. I know the current MEV-Boost already handles this, but it is useful for epbs and future iterations as well.